src/main/java/org/jabref/logic/importer/Importer.java

```diff
@@ -77,6 +77,12 @@ public boolean isRecognizedFormat(String data) throws IOException {
      * <p>
      * If importing in a specified format and an empty library is returned, JabRef reports that no entries were found.
      * <p>
+     * If your format is binary-based (PDF, ZIP-based, or others), then you should not solely override this method.
+     * For binary formats do this:
+     * 1. Throw {@link UnsupportedOperationException} in this method.
+     * 2. Override the method {@link Importer#importDatabase(Path)}.
```

Comment:
Follow-up: convert to `<ol>` or convert the whole JavaDoc to Markdown ( `///`). Otherwise, this will not be rendered as list, but as text